Cargo.toml

```diff
@@ -206,10 +208,11 @@ tempfile = "3.10.1"
 terrors = "0.3.0"
 thiserror = "1.0"
 threadpool = "1.8"
-tokio = { version = "1.37", features = ["full"] }
+tokio = { version = "1.*", features = ["full"] }
```

Comment:
We should pin tokio.
This is fine for scaffolding the PR.
Imagine two versions of tokio, the huge compile time...
I know that we're somewhat between a rock and a hard place with https://github.com/tokio-rs/tracing-opentelemetry/issues/170

Reply:
Oh I think I forgot to fix this.  I changed it while I was trying to iron out the dependency mixups.